target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -229,6 +234,11 @@ case "$FIRMWARE" in
 		caldata_valid "202f" || caldata_extract "ART" 0x5000 0x2f20
 		ath10k_patch_mac $(macaddr_add $(get_mac_binary "/sys/bus/i2c/devices/0-0050/eeprom" 0x66) +3)
 		;;
+	mikrotik,hap-ac2)
+		wdata="/sys/firmware/mikrotik/hard_config/wlan_data"
+		( [ -f "$wdata" ] && caldata_sysfsload_from_file "$wdata" 0x8000 0x2f20 ) || \
+		( [ -d "$wdata" ] && caldata_sysfsload_from_file "$wdata/data_2" 0x0 0x2f20 )
+		;;
```

Comment:
Likewise; this belongs to the subtarget environment. See target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/11-ath10k-caldata for styling